.gitignore

```
@@ -41,6 +41,11 @@ npm-debug.log
 # Dialyxir output
 /priv/plts/
 
+# Gateway ephemeral files
+/priv/gateway-conf/id
+/priv/gateway-conf/logs/
+/priv/gateway-conf/token
```

Maybe it's better just to use one `gateway` folder and then add subfolders to it so that we don't list out all files the gateway might use?
```suggestion
/priv/gateway
```

---

We might want to make sure that `/priv` would not be distributed with all elixir releases, it's is packaged by default when you run `mix release`. Can we add it to `.dockerignore`?

---

I want some of the files in `priv/gateway-conf` to not be in .gitignore

---

namely, the development private key for the gateway wireguard interface.

---

You can do 
```gitignore
/priv/gateway/*
!/priv/gateway/my_file
```

---

:+1: done that

but `priv` can't go into `.dockerignore`, the Dockerfiles use that directory during the build step build(although the final image doesn't include it so I don't think it's that bad)